app/src/main/java/bankingapp/App.java

```diff
@@ -1,5 +1,5 @@
 /*
- * This Java source file was generated by the Gradle 'init' task.
```

Comment:
I recommend adding a LICENSE file at the top level later on. Otherwise, github defaults to MIT.